artiq/coredevice/ad9910.py

```diff
@@ -64,6 +65,9 @@
 # Default profile for RAM mode
 _DEFAULT_PROFILE_RAM = 0
 
+AD9910_REG_FTW = _AD9910_REG_FTW
+AD9910_REG_PROFILE0 = _AD9910_REG_PROFILE0
+AD9910_REG_PROFILE7 = _AD9910_REG_PROFILE7
```

Comment:
Remove the ``_`` in the first place (in a separate PR of course) or just import with ``_``.

Reply:
done